README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+## General Bio
+Hello, my name is Jackson (though I go by Jay) Gibbs. I am a fourth year quadruple major studying:
+* East Asian Languages and Civilizations 
+* Public Policy
+* Political Science
+* Comparitive Race and Ethnic Studies
+
+### Why I am taking the course
+I've always had an interest in the data side to Public Policy and social science. That being said, I am *very* inexperienced when it comes to using R and Stata for more than the basic regressions and things Public Policy majors do in some of the required classes. Given that, a large part of my interest is in **learning the quantitative side** as oppossed to the qualitative side of data analysis. 
+
+### Where I hope to use what I learn
+One main place I have had to do a lot of analysis recently is for my BA thesis looking at Black female informal labor markets. While there is a good deal of analysis in that papper, every single bit of it (with the exception of one regression) was done using Excel. I got a chance to present it at the BA symposium a few weeks back:
+
+![image of Jay Gibbs presenting his Public Policy thesis](https://raw.githubusercontent.com/jaycgibbs/hw01/master/88236501_3892564654087317_4261089663416008704_n.jpg)
```

Comment:
You can also link using relative file paths (i.e. for a picture that's also committed, but lives primarily on your computer).

Reply:
I think that your photo is great, but we need to scroll to look at it fully. It would be nice if it were a bit smaller. 
I will not tell you how to do it because I myself tried so many ways with the markdown code to reduce the size of my photo and it did not work. I ended up using a HTML code (found the solution online), which helped reduce the image size.... but as one of the TAs told me, this homework is to practice the markdown syntax. So I'm hoping we'll learn about this soon :) 
One thing that I do know is that if your photo is in your github repository already, you can just put the path like this: `![Title of your photo](name_of_file.png)`